engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowOutputsActor.scala

```diff
@@ -68,9 +68,7 @@ class CopyWorkflowOutputsActor(workflowId: WorkflowId, override val ioActor: Act
           s" as multiple files will be copied to the same path: \n${formattedCollidingCopyOptions.mkString("\n")}")}
 
     val copies = outputFilePaths map {
-      case (srcPath, dstPath) => 
-        dstPath.createDirectories()
```

Comment:
are we sure we don't want to be creating destination directories?  or is it implied in the copy command?

cc @cjllanwarne

Reply:
Directory creation happens as a part of copy command